### PR TITLE
Add postgres.properties template to README and allow users to omit fhir truststore

### PR DESCRIPTION
**Pull request by lmsurpre**
Omitting the truststore will result in the client using the default java
cacerts truststore which works will any of our cloud deploys.

I also allowed my editor to apply the "remove trailing whitespace"
action

Signed-off-by: Lee Surprenant <lmsurpre@us.ibm.com>

### CODE REVIEW

File: fhir-bucket/src/main/java/com/ibm/fhir/bucket/client/FHIRBucketClient.java [hunk 1]
```diff
@@ -124,13 +124,16 @@ public long getKeepAliveDuration(HttpResponse response, HttpContext context) {
                 return 60000*60;
             }
         };
-        
+
         try {
             // SSLContext sslContext = SSLContexts.custom().build();
```
Comment (prb112) [inline fhir-bucket/src/main/java/com/ibm/fhir/bucket/client/FHIRBucketClient.java:129]: dead code?

File: fhir-bucket/src/main/java/com/ibm/fhir/bucket/client/FHIRBucketClient.java [hunk 2]
```diff
@@ -124,13 +124,16 @@ public long getKeepAliveDuration(HttpResponse response, HttpContext context) {
...
-            
+
             SSLContextBuilder sslContextBuilder = SSLContextBuilder.create();
             // sslContextBuilder.loadKeyMaterial(new File(keystoreFilename), keystorePass.toCharArray(), keyPass.toCharArray());
```
Comment (prb112) [inline fhir-bucket/src/main/java/com/ibm/fhir/bucket/client/FHIRBucketClient.java:132]: dead code?

**Review by prb112 — APPROVED** [review]
LGTM

**Review by punktilious — APPROVED** [review]
LGTM. Nice catch with that truststore/keystore thing

**Pull request merged**